libs/utils/languages.js

```diff
@@ -0,0 +1,22 @@
+export default {
+  en: {
+    ietf: 'en',
```

Comment:
do we have use-cases where ietf is different than the lang ?

Reply:
Good point! - For the lang probably not and for the lang-region very rarely

Reply:
I'll remove the unnecessary usages.